### DISCUSSION TITLE
What should we do about the EIGHT new addons (and counting) this release?

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#3289] [Discussion]
The "new" addon group fills the whole settings page (screenshot from master):
![image](https://user-images.githubusercontent.com/17484114/129490959-66417787-115f-46f0-becf-d9d80a178384.png)

Also, so many addons I would classify as "minor". Most, actually.
Only 2 amazing ones imo are backpack naming and paint by default.
IMO those 2 should stay visible at the top the whole time. The other 6+ could be in another group that's collapsed by default (or maybe expanded only on first load of settings page after update)

**Comment by mxmou** [comment]
One option is to implement the featured tag and only show featured addons in the "new" group, with a "show more" button to show the rest.

**Reply by TheColaber** [reply]
I had this idea too and told it to wl yesterday on discord and he said "¯\\\_(ツ)_/¯"

**Reply by WorldLanguages** [reply]
A "show more" button can be a bit complex because of the virtual DOM optimizations I made some time ago. It would be way easier to have separate "Featured new addons" and "New addons" groups, and have the latter be collapsed by default.

**Reply by WorldLanguages** [reply]
https://github.com/ScratchAddons/ScratchAddons/pull/3302

**Reply by TheColaber** [reply]
I think a load more button wouldn't be too bad. There would just be a new variable to the manifests that should not show the addon if the user hasn't pressed the button.

**Comment by z-anonymouz** [comment]
I think you should try to add this extension to Microsoft Edge Addons. Microsoft Edge currently supports Chrome extensions but they will change it soon.

**Reply by z-anonymouz** [reply]
Soon Edge will not be able to use Chrome extensions. The reason why Edge can use Chrome extensions in the first place is that Edge is built on a Chromium source. I heard Microsoft will stop using Chromium. I use Edge so I would like to still use Scratch Addons.

**Reply by Secret-chest** [reply]
What???

**Reply by Samq64** [reply]
1. Where are you getting your information?
2. Why would Edge stop using chromium if they just switched to chromium few years ago?
3. Why are you posting here?

**Reply by WorldLanguages** [reply]
> I heard Microsoft will stop using Chromium.

Nah.
